### PR TITLE
Replace Cognito with Keycloak and remove cognito

### PR DESCRIPTION
**Pull request by aneelac22**

### CODE REVIEW

File: src/jwt/__mocks__/keycloak-js.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Keycloak {
     this.responseMode = 'fragment';
     this.responseType = 'code';
     this.flow = 'standard';
-    this.clientId = ITLessKeycloak() ? 'console-dot' : 'cloud-services';
+    this.clientId = ITLess() ? 'console-dot' : 'cloud-services';
```
Comment (florkbr) [inline src/jwt/__mocks__/keycloak-js.ts:35]: Not blocking - but we have this exact same check/string here: https://github.com/RedHatInsights/insights-chrome/pull/2708/files#diff-345249b2500e3d310d8235c95d24191bb7a2dc823e0a6f4d2a8e4da2b45a697aR68

Might be worth creating a clientId helper so we don't have to change it in multiple locations in the future.

Comment (aneelac22) [reply src/jwt/__mocks__/keycloak-js.ts:35]: Yeah, I agree, will do that when Martin's changes are merged

File: src/auth/index.ts [hunk 2]
```diff
@@ -5,8 +5,6 @@ import { ChromeUser } from '@redhat-cloud-services/types';
 import { Store } from 'redux';
 
 import * as jwt from '../jwt/jwt';
-import { createUser, getTokenWithAuthorizationCode } from '../cognito/auth';
```
Comment (florkbr) [inline src/auth/index.ts:8]: FWIW these utils will be refactored in Martin's PR here: https://github.com/RedHatInsights/insights-chrome/pull/2702

File: src/jwt/jwt.ts [hunk 3]
```diff
@@ -171,80 +169,52 @@ export const init = (options: JWTInitOptions, configSsoUrl?: string) => {
...
-  } else {
-    return Promise.resolve(platformUrl(options.routes ? options.routes : DEFAULT_SSO_ROUTES, configSsoUrl)).then((ssoUrl) => {
-      //constructor for new Keycloak Object?
-      options.url = ssoUrl;
-      options.clientId = itLessKeycloakEnv ? 'console-dot' : 'cloud-services';
-      options.realm = 'redhat-external';
+  return Promise.resolve(platformUrl(options.routes ? options.routes : DEFAULT_SSO_ROUTES, configSsoUrl)).then((ssoUrl) => {
+    //constructor for new Keycloak Object?
+    options.url = ssoUrl;
+    options.clientId = ITLess() ? 'console-dot' : 'cloud-services';
```
Comment (florkbr) [inline src/jwt/jwt.ts:175]: Another spot we can de-dupe. Not blocking and we will follow up.

**Review by florkbr — APPROVED** [review]

**Comment by florkbr** [comment]
Tests failures seem unrelated to these changes:
```log
10:00:31 =========================== short test summary info ============================
10:00:31 FAILED tests/test_chrome_components.py::test_favorites_nav - wait_for.TimedOu...
10:00:31 FAILED tests/test_chrome_components.py::test_app_filter_search[search_terms0]
10:00:31 = 2 failed, 3 passed, 371 deselected, 2 xfailed, 7 warnings in 114.87s (0:01:54) =
10:00:31 There is a test failing in your suite. To debug it:
10:00:31 	See https://consoledot.pages.redhat.com/docs/dev/operating-your-app/testing-iqe/smoke-more.html#_testing_code_changes_to_a_smoke_test for more details on how to deploy to ephemeral environments and debug your tests
10:00:31 ��� ERROR: plugin platform_ui: sequential test run hit errors (exit code 1)
10:00:31 ��� ERROR: plugin platform_ui: tests failed
```

**Pull request merged**